src/Page/Editor.purs

```diff
@@ -48,32 +48,31 @@ type State =
 type Input =
   { slug :: Maybe Slug }
 
-type ChildQuery m = F.Query Query TagInput.Query Unit EditorFields m
+type ChildSlots =
+  ( formless :: F.Slot EditorFields (Const Void) FormChildSlots Article Unit )
```

Comment:
Formless is the component that has undergone the greatest number of changes as part of the Halogen 5 transition. For the full details, [see the Formless library](https://github.com/thomashoneyman/purescript-halogen-formless). If you have written a form and you want to extend the form's state, query algebra, actions, messages, child components, or anything else, that information no longer leaks out to the parent component but instead is handled entirely within Formless. Accordingly, any extensions you want to make to the component will show up in its slot type.

The main thing that affects us here is that the Formless slot type now takes:

- Your form type being run in Formless (here, `EditorFields`)
- Any queries you wish to extend the component with (here, none, represented with `Const Void`
- Any child components the form will have (here, `FormChildSlots`, defined at the end of the module)
- Any messages the component will output (here, we'll get an `Article` on successful submission)
- The form slot address